### PR TITLE
Fix connection problem

### PR DESCRIPTION
**Pull request by Debora38**
Hi @rockem , I encountered few errors with the previous code, fixed with the below changes

Screenshot 2022-01-31 at 12 08 17

Screenshot 2022-01-31 at 12 07 46

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,9 +2,9 @@ const core = require('@actions/core');
 const opsgenie = require('opsgenie-sdk');
 
 
-connectionDetails = {'api_key': core.getInput('api_key')}
+const connectionDetails = {'api_key': core.getInput('api_key')}
 if (core.getInput('using_eu_url')) {
```
Comment (rockem) [inline index.js:6]: I think this is wrong as well as ```core.getInput('using_eu_url')``` is a string

Comment (Debora38) [reply index.js:6]: We tested this action and got the errors I screenshot in the description of the PR. After these changes the action worked. We'll be using a local action now, but I'll leave my suggestions in this PR if you want to replicate the error and look into them ;)

Comment (rockem) [reply index.js:6]: @Debora38 I'm ok with the changes and would like to push it. Don't you agree on the condition?

Comment (Debora38) [reply index.js:6]: @rockem sorry what condition?

**Comment by rockem** [comment]
@Debora38 Hi. I'm adding tests (so it won't happen again) and will release a fixed version soon

**Comment by rockem** [comment]
@Debora38 This issue is now fix on either v1.2.2 or v1. Please update me if it still has any issue

**Pull request closed without merging**